homeassistant/components/google_assistant/trait.py

```diff
@@ -517,7 +517,10 @@ class TemperatureSettingTrait(_Trait):
         climate.STATE_HEAT: 'heat',
         climate.STATE_COOL: 'cool',
         climate.STATE_OFF: 'off',
-        climate.STATE_AUTO: 'heatcool',
+        climate.STATE_AUTO: 'auto',
```

Comment:
The climate component will use auto as to be the one where we have a temperature range, and it will either heat or cool to stay within that range. This behavior is equivalent to the `heatcool` description of Actions on Google.

https://developers.google.com/actions/smarthome/traits/temperaturesetting

 - `heatcool` If the device supports maintaining heating/cooling to target a range.
 - `auto` If the device supports an "automatic" mode where the temperature is set based on a schedule, learned behavior, AI, or some other related mechanism.

Reply:
reverted heatcool -> auto change in 385f029